src/app/route/index.js

```diff
@@ -63,6 +64,10 @@ export class AppRouter extends Component {
                     component: <Route path="/category" component={ CategoryPage } />,
                     position: 20
                 },
+                {
+                    component: <Route path="/search/:query/" component={ SearchPage } />,
```

Comment:
Are you sure you want this to be a URL param? Maybe we should use get params instead.